RELEASE/scripts/autoscend/iotms/mr2023.ash

```diff
@@ -989,3 +995,36 @@ void auto_useWardrobe()
 	// let maximizer handle if any of it is worth equipping
 	use($item[wardrobe-o-matic]);
 }
+
+boolean fightFlamingLeaflet() {
```

Comment:
I know a lot of the code is inconsistent about this, but all the other leaves methods have the `auto_` prefix, might as well do it for these ones too.